Makefile

```diff
@@ -226,6 +226,13 @@ urn:
 	@printf "⚱\n\n";
 	@$(MAKE) run;
 
+docker-build:
+	docker build -t wp-calypso .
+
+docker-run:
+	docker run -it --name wp-calypso --rm -p 80:3000 -e NODE_ENV='wpcalypso' -e CALYPSO_ENV='wpcalypso' wp-calypso
```

Comment:
that sounds fine. you can insert it as a line above 233 - not in the best spot to make a commit at the moment